apps/api/src/app/events/e2e/digest-events.e2e.ts

```diff
@@ -557,7 +558,7 @@ describe('Trigger event - Digest triggered events - /v1/events/trigger (POST)',
       _environmentId: session.environment._id,
       channel: StepTypeEnum.IN_APP,
       _templateId: template._id,
-    });
+    } as any);
```

Comment:
Is there any thing here we can do to avoid `any` usage? 
Or at least add eslint-ignore for this line so we reduce the amounts of warnings in the project :)

Reply:
Well, I was not sure what other object would be better here..